### ISSUE TITLE
sql db issue 

### ISSUE THREAD

**Issue by joetricot** [joetricot/Civic_Solutions#1]
My CRUD finally seems to be working for the most part...
However, when I login as a different user, the same reports that I generated through a different account still show up.
I checked my SQL database and it seems that all data being entered in the table have the same user_id of 3.
I walked through the delorean movies and todo apps to see if I made any errors, but can't catch anything yet.
This is the last thing to get done before I focus on my API which I know will be a lot more work.

**Comment by drew-mahrt** [comment]
Are you checking your data with psql in the terminal?

**Comment by joetricot** [comment]
Yes...that's how I know that data being entered has a user_id of 3
